src/components/color-picker-hex-input/color-picker-hex-input.tsx

```diff
@@ -67,33 +67,31 @@ export class ColorPickerHexInput {
   //--------------------------------------------------------------------------
 
   /**
-   * When false, empty color (null) will be allowed as a value. Otherwise, a color value is always enforced by the component.
+   * When `false`, an empty color (`null`) will be allowed as a `value`. Otherwise, a color value is enforced on the component.
    *
-   * When true, clearing the input and blurring will restore the last valid color set. When false, it will set it to empty.
+   * When `true`, clearing the input and blurring will restore the last valid `value`. When `false`, it will set the `value` to empty.
```

Comment:
These two lines pretty much say the same thing. Maybe we can combine them into something like

> When `true`, a color value is enforced, and clearing the input or blurring will restore the last valid `value`.  When `false`, an empty color (`null`) will be allowed as a `value`.